src/applications/ivc-champva/shared/utilities.js

```diff
@@ -30,3 +30,27 @@ export function applicantWording(
 
   return isPosessive ? `${retVal}’s` : retVal;
 }
+
+// Turn camelCase into capitalized words ("camelCase" => "Camel Case")
+export function makeHumanReadable(inputStr) {
+  return inputStr
+    .match(/^[a-z]+|[A-Z][a-z]*/g)
+    .map(word => word[0].toUpperCase() + word.substr(1).toLowerCase())
+    .join(' ');
+}
+
+/**
+ * Evaluate the `depends` func of each provided page to determine
+ * its value.
+ * @param {object|list} pages A subset of pages within the form
+ * @param {object} data formData used in `depends` calculations
+ * @param {number} index Optional argument to pass to `depends` if evaluating list and loop page `depends`
+ * @returns A filtered list of pages where `depends` was true
+ */
+export function getConditionalPages(pages, data, index) {
+  const tmpPg =
+    typeof pages === 'object' ? Object.keys(pages).map(pg => pages[pg]) : pages;
+  return tmpPg.filter(
+    pg => pg.depends === undefined || pg?.depends({ ...data }, index),
+  );
+}
```

Comment:
Just moved these into the shared utilities file because that makes a lot more sense.